internal/service/resources/api/openapi.yaml

```diff
@@ -1067,3 +1066,42 @@ components:
       - managementInterfaceId
       - pkNotificationField
       - alarmAdditionalFields
+
+    InventoryChangeNotification:
+      description: Information about an inventory change notification
+      type: object
+      properties:
+        notificationId:
+          type: string
+          format: uuid
+          description: |
+            A unique identifier to represent this notification event
+        consumerSubscriptionId:
+          type: string
+          description: |
+            The value provided by the consumer in the subscription
+        notificationEventType:
+          type: integer
+          description: |
```

Comment:
why don´t you add an enum here so that the check is performed by the middleware?

Reply:
I could add an enum here, but it is not needed for validation as this object is outbound only.  The user never sends us one of these.   I'll change it.

Reply:
Fixed